lib/deep-speech-interface.js

```diff
@@ -84,68 +84,80 @@ class DeepSpeechInterface {
+        '--kenlm_bins', this._binDir,
+        '--arpa_order', '5',
+        '--max_arpa_memory', '50%',
+        '--arpa_prune', '0|0|1',
+        '--binary_a_bits', '255',
+        '--binary_q_bits', '8',
+        '--binary_type', 'trie',
         '--discount_fallback',
-        '--text', localLMTxt,
-        '--arpa', localLMArpa,
       ]
     );
 
     if (this._debug) {
-      console.debug('lmplz: status:', lmplz.status);
-      console.debug('lmplz: stdout:', lmplz.stdout.toString());
-      console.debug('lmplz: stderr:', lmplz.stderr.toString());
+      console.debug('generate_lm.py: status:', generateLm.status);
+      console.debug('generate_lm.py: stdout:', generateLm.stdout.toString());
+      console.debug('generate_lm.py: stderr:', generateLm.stderr.toString());
     }
 
-    const buildBinary = spawnSync(
-      path.join(this._binDir, 'build_binary'),
+    const generateScorer = spawnSync(
+      path.join(this._binDir, 'generate_scorer_package'),
       [
-        '-a', '255',
-        '-q', '8',
-        'trie',
-        localLMArpa,
-        localLMBinary,
+        '--lm', path.join(this._modelsDir, 'lm.binary'),
+        '--vocab', lmText,
+        '--package', path.join(this._modelsDir, 'kenlm.scorer'),
+        '--default_alpha', '0.931289039105002',
+        '--default_beta', '1.1834137581510284',
+        '--force_utf8', 'true',
+        // https://github.com/mozilla/DeepSpeech/issues/3205
+        '--alphabet', 'fake',
       ]
     );
 
     if (this._debug) {
-      console.debug('build_binary: status:', buildBinary.status);
-      console.debug('build_binary: stdout:', buildBinary.stdout.toString());
-      console.debug('build_binary: stderr:', buildBinary.stderr.toString());
+      console.debug(
+        'generate_scorer_package: status:',
+        generateScorer.status
+      );
+      console.debug(
+        'generate_scorer_package: stdout:',
+        generateScorer.stdout.toString()
+      );
+      console.debug(
+        'generate_scorer_package: stderr:',
+        generateScorer.stderr.toString()
+      );
     }
 
     if (this._debug) {
       console.debug(`Loading model from ${this._modelsDir}`);
     }
 
-    const modelJson =
-      JSON.parse(fs.readFileSync(path.join(this._assetsDir, 'info.json')));
-
     const extension = process.arch === 'x64' ? 'pbmm' : 'tflite';
     this._model = new Ds.Model(
-      path.join(this._assetsDir, `output_graph.${extension}`),
-      500
+      path.join(this._assetsDir, `deepspeech-model.${extension}`)
     );
 
     if (this._debug) {
       console.debug(`Enabling decoder on model ${this._modelsDir}`);
     }
 
-    this._model.enableDecoderWithLM(
-      path.join(this._modelsDir, 'local_lm.binary'),
-      path.join(this._modelsDir, 'local_lm.trie'),
-      modelJson.parameters.lmAlpha,
-      modelJson.parameters.lmBeta
+    this._model.enableExternalScorer(
+      path.join(this._modelsDir, 'kenlm.scorer')
     );
+    this._model.setBeamWidth(500);
```

Comment:
This is preserving what was previously used, but I'm not sure if it's still correct (or necessary).

Reply:
Again, there is à default one set in the model, you should not force a new value without knowing what you do